charts/pega/values-large.yaml

```diff
@@ -59,8 +59,7 @@ global:
     # CUSTOM CONNECTION PROPERTIES
     # Add a list of ; delimited connections properties. The list must end with ;
```

Comment:
Let's try this slight edit....is it `` or ""?
Add a list of semi-colon-delimited required connection properties to pass as part of the connection. 
The list must end with ;. For example, to set a custom authentication using Azure Managed Identity
Set  `connectionProperties: "Authentication=ActiveDirectoryMSI;msiClientId=1234;"`